server/app/models/stack.rb

```diff
@@ -11,6 +11,8 @@ class Stack
   has_many :stack_revisions, dependent: :destroy
   has_many :stack_deploys, dependent: :destroy
   has_many :grid_services
+  has_many :volumes
+  embeds_many :external_volumes
```

Comment:
Ok, but somehow this smells a bit .. could we map volumes always the same way (and just have extra attributes set whenever needed).